apps/wallet/src/background/Permissions.ts

```diff
@@ -275,6 +276,29 @@ class Permissions {
         }
     }
 
+    public async ensurePermissionAccountsUpdated(accounts: Account[]) {
+        const allPermissions = await this.getPermissions();
+        const availableAccountsIndex = accounts.reduce<Record<string, boolean>>(
+            (acc, { address }) => {
+                acc[address] = true;
+                return acc;
+            },
+            {}
+        );
+        Object.entries(allPermissions).forEach(
+            async ([origin, { accounts, allowed }]) => {
+                if (allowed) {
+                    const accountsToDisconnect = accounts.filter(
+                        (anAddress) => !availableAccountsIndex[anAddress]
+                    );
+                    if (accountsToDisconnect.length) {
+                        await this.delete(origin, accountsToDisconnect);
```

Comment:
does this need to be done serially, e.g. delete permissions for site 1 -> delete permissions for site 2 -> etc. or can we do them in parallel?

Reply:
Fair enough, just wanted to check :)